packages/web/src/components/Grid/_Grid.scss

```diff
@@ -13,6 +13,10 @@
 
 @include tools.equal-columns(theme.$grid-equal-columns, theme.$breakpoints);
 
+.Grid--narrow {
+    grid-template-columns: repeat(theme.$grid-columns, 1fr);
```

Comment:
Could we have this in a standalone commit please? It's not connected to token changes.